vectorbendertransformers.py

```diff
@@ -111,25 +111,19 @@ def __init__(self, pairsLayer, restrictToSelection):
 
 
         """
-
```

Comment:
Revert these unrelated/unnecessary changes (for the whole file).

If you wish to improve the style, that's welcome, but should be part of another PR (so that this one really stays about porting to QGIS3)

Reply:
I tried to revert, I don't know if it's good now, because of number of spaces characters.